solr/core/src/java/org/apache/solr/core/backup/BackupManager.java

```diff
@@ -323,15 +323,13 @@ private void uploadConfigToConfigRepo(ConfigSetService configSetService, URI sou
             is.readBytes(arr, 0, (int) is.length());
             if(filePath.startsWith("/")) {
```

Comment:
I think you misunderstand what I was trying to say.  Can you see how that leading "/" got there in the first place?  (hint; it's earlier in this very method).  So my advise is to not put it there in the first place (instead of always removing later).